gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -150,6 +152,12 @@ def construct_argparser(self) -> argparse.ArgumentParser:
             default=3,
             help="Number of times to retry cluster connection if it fails.",
         )
+        parser.add_argument(
+            "--run_low_quality",
```

Comment:
I would recommend changing the name to something like `run_single_trial` or `skip_trials` with low quality mentioned in the description.

Reply:
Updated, thanks.